misk-rate-limiting-bucket4j-dynamodb-v1/src/main/kotlin/misk/ratelimiting/bucket4j/dynamodb/v1/DynamoDbV1BucketPruner.kt

```diff
@@ -0,0 +1,80 @@
+package misk.ratelimiting.bucket4j.dynamodb.v1
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
+import com.amazonaws.services.dynamodbv2.document.DynamoDB
+import com.amazonaws.services.dynamodbv2.document.Expected
+import com.amazonaws.services.dynamodbv2.document.spec.DeleteItemSpec
+import com.amazonaws.services.dynamodbv2.document.spec.ScanSpec
+import com.amazonaws.services.dynamodbv2.model.ReturnValue
+import io.micrometer.core.instrument.MeterRegistry
+import wisp.logging.getLogger
+import wisp.ratelimiting.RateLimitPrunerMetrics
+import wisp.ratelimiting.bucket4j.Bucket4jPruner
+import wisp.ratelimiting.bucket4j.ClockTimeMeter
+import java.time.Clock
+import kotlin.system.measureTimeMillis
+
+class DynamoDbV1BucketPruner @JvmOverloads constructor(
```

Comment:
who's calling this?

Reply:
Currently nobody, it needs to be implemented in services via some recurring task mechanism, e.g. misk-cron